dbt_server/logging.py

```diff
@@ -104,77 +109,23 @@ def to_json(self):
         return json.dumps(self.__dict__)
 
 
+# TODO: Make this a contextmanager
 class LogManager(object):
     def __init__(self, log_path):
-        from dbt_server.services import filesystem_service
-
+        self.name = str(uuid.uuid4())
         self.log_path = log_path
-
         filesystem_service.ensure_dir_exists(self.log_path)
-
-        logs_redirect_handler = logbook.FileHandler(
-            filename=self.log_path,
-            level=logbook.DEBUG,
-            bubble=True,
-            # TODO : Do we want to filter these?
-            filter=self._dbt_logs_only_filter,
+        logger_config = LoggerConfig(
+            name=self.name,
+            line_format=LineFormat.DebugText,
+            level=EventLevel.INFO,
+            use_colors=True,
+            output_file_name=log_path,
```

Comment:
For my understanding, We are writing the logs to a local file then `stream-log` endpoint gonna read that file and send SSE to WS controller?

Reply:
Yes, that is currently how it's working! This is just bringing us to parity with how the server logging worked before, we aren't set on this implementation.

Reply:
My goal here would be: finding a straightforward way to get this working for now, while preserving future options where we push events elsewhere / elsehow. Sounds like this approach fits the bill